lib/TPP/TileConsumerAndFuseProducers.cpp

```diff
@@ -704,72 +771,27 @@ struct TileConsumerAndFuseProducers
       (void)applyPatternsAndFoldGreedily(getOperation(), std::move(patterns));
     }
 
-    func::FuncOp func = getOperation();
-    IRRewriter rewriter(&getContext());
-
-    // Set to keep track of fused ops.
-    llvm::SmallDenseSet<Operation *> fusedOps;
-
-    SmallVector<linalg::LinalgOp> linalgContractionOperations;
-    func->walk([&](linalg::LinalgOp linalgOp) {
-      if ((isConvolutionLike(linalgOp) ||
-           succeeded(linalgx::utils::isContraction(linalgOp))) &&
-          linalgOp.hasTensorSemantics())
-        linalgContractionOperations.push_back(linalgOp);
-    });
-
-    // Reverse the ops. Start from the bottom to increase
-    // the fusion boundaries.
-    std::reverse(linalgContractionOperations.begin(),
-                 linalgContractionOperations.end());
-    llvm::SmallDenseSet<Operation *> visitedConsumers;
-    llvm::SmallDenseSet<Operation *> fusionRoots;
-
-    // Compute the tile sizes for each contraction operations or
-    // use the default one.
-    llvm::DenseMap<Operation *, SmallVector<OpFoldResult>> defaultTiles;
-    for (auto contractionOp : linalgContractionOperations) {
-      auto tiles = getDefaultTileSizes(contractionOp, this->tileSizes);
-      if (failed(tiles)) {
-        LLVM_DEBUG(llvm::dbgs() << "Failed to compute default tile sizes for: "
-                                << contractionOp << "\n");
-        return;
+    int fixedPoint = 3;
```

Comment:
Any test covers/showcases this new approach?

Reply:
I can extract a smaller test case from MHA